### ISSUE TITLE
fatal error has been detected by the Java Runtime Environment

### ISSUE THREAD

**Issue by jsjain** [corretto/corretto-11#381]
### Describe the bug
I am running my questDB server with `JRE version: OpenJDK Runtime Environment Corretto-11.0.22.7.1 (11.0.22+7) (build 11.0.22+7-LTS)`, under heavy write operations my server is crashing with error 

```
Dec 23 17:41:42 ip-10-1-0-188.ap-south-1.compute.internal java[39732]: #
Dec 23 17:41:42 ip-10-1-0-188.ap-south-1.compute.internal java[39732]: # A fatal error has been detected by the Java Runtime Environment:
Dec 23 17:41:42 ip-10-1-0-188.ap-south-1.compute.internal java[39732]: #
Dec 23 17:41:42 ip-10-1-0-188.ap-south-1.compute.internal java[39732]: #  SIGSEGV (0xb) at pc=0x00007f2c7acba0c9, pid=39732, tid=39766
```

### Expected behavior
To not crash

### Platform information
`OS: Amazon Linux 2`
`AWS m6a.2xlarge server`
`Corretto-11.0.22.7.1 (11.0.22+7) (build 11.0.22+7-LTS)`

### Additional context
Attached crash dump generated by JVM
[hs_err_pid39732.log](https://github.com/user-attachments/files/18233424/hs_err_pid39732.log)

For VM crashes, please attach the error report file. By default the file name is `hs_err_pidpid.log`, where pid is the process ID of the process.

**Comment by cost0much** [comment]
Hello, thank you for contacting the Corretto team! When did you first start seeing this issue? Also, have you contacted QuestDB yet, and were they able to provide any insight on the matter?

**Comment by chadrako** [comment]
I see that you are using an old version of Corretto 11 (11.0.22.7.1). Does the issue reproduce on the most recent version of Corretto 11 (11.0.25.9.1)?

**Comment by jsjain** [comment]
I have upgraded the coretto version, will confirm if it comes again

**Comment by mharish12** [comment]
This Happened for me when I enabled `Scheduled` debugger, It got resolved when i removed it.

**Comment by CrimsonPlumIntoSnow** [comment]
Yes, I encountered the same issue. The detailed process is as follows: Added a new piece of code in a method, and then `recompile` the Java file in IDEA. After the class reloading was completed, I triggered the method and confirmed through logs that the newly added code was executed. Then, I added a breakpoint in this new section of code, triggered the method again, and the JVM crashed.

The crash information is as follows
<img width="1657" height="677" alt="Image" src="https://github.com/user-attachments/assets/f284b85d-3dfd-4ef5-b3a0-84f03863c423" />

**Comment by cost0much** [comment]
@CrimsonPlumIntoSnow I suggest you open a separate issue, as I believe these aren't the same issue. The original issue described relates to a questDB application on an AWS linux host environment. Your issue concerns following specific steps on a IDEA application in a Windows environment. Closing.
